.github/workflows/build_and_test_workflow.yml

```diff
@@ -138,7 +138,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest]
```

Comment:
Do we want to do the tests that are done in windows or not?

I understand from this that several windows tests are not going to be run.